pkg/sql/sem/builtins/generator_builtins.go

```diff
@@ -46,7 +46,7 @@ import (
 var _ eval.ValueGenerator = &seriesValueGenerator{}
 var _ eval.ValueGenerator = &arrayValueGenerator{}
 
-func initGeneratorBuiltins() {
+func init() {
```

Comment:
This is an okay change; but it probably should be done in a separate PR if you feel
strongly about this cleanup.